spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultSchedulerService.java

```diff
@@ -137,11 +142,19 @@ public void schedule(String scheduleName, String taskDefinitionName, Map<String,
 		ScheduleRequest scheduleRequest = new ScheduleRequest(revisedDefinition, taskDeploymentProperties,
 				deployerDeploymentProperties, commandLineArgs, scheduleName, getTaskResource(taskDefinitionName));
 		this.scheduler.schedule(scheduleRequest);
+		this.auditRecordService.recordScheduleCreate(scheduleRequest);
 	}
+	public static final String STREAM_DEFINITION_DSL_TEXT = "streamDefinitionDslText";
```

Comment:
Please see https://github.com/spring-cloud/spring-cloud-dataflow/pull/2485/files#r223937332

Reply:
Yes - will be done as part of https://github.com/spring-cloud/spring-cloud-dataflow/issues/2477 I wanted to do it as part of this PR but as it got a big bigger...